packages/paypal-commerce-integration/src/paypal-commerce-ratepay/paypal-commerce-ratepay-payment-strategy.spec.ts

```diff
@@ -51,15 +52,21 @@ describe('PayPalCommerceAlternativeMethodRatePayPaymentStrategy', () => {
         paypalcommerceratepay: paypalCommerceRatePayMethodsOptions,
     };
 
+    const loadingIndicator = new LoadingIndicator();
```

Comment:
It might be better to define `loadingIndicator` in the top of parent describe method (line 30), and set the value of the variable inside `beforeEach` method as `paymentIntegrationService` and `paypalCommerceIntegrationService`

Reply:
fixed